SECURITY.md

```diff
@@ -0,0 +1,39 @@
+# Security Policy
+
+In **Amplica Labs** we take the security of our open-source software seriously. We
+appreciate the efforts of security researchers and the broader community in helping us maintain a
+secure environment for our users. If you discover a security vulnerability, we kindly request that
+you report it to us privately before disclosing it publicly. This allows us to address the issue
+promptly and protect our users.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report the vulnerability via [https://github.com/LibertyDSNP/frequency/security](https://github.com/LibertyDSNP/frequency/security).
+
+Or alternatively, you can send email to [security@frequency.xyz](mailto:security@frequency.xyz).  If
+possible, encrypt your message with our PGP key; you can download it from [MIT's key server](<replace url after upload>)
+or copy from below.
+
+You should receive a response within 48 hours. If for some reason you do not, please follow up via
+email to ensure we received your original message.
+
+## Report details
+Please include the requested information listed below (as much as you can provide) to help us better
+understand the nature and scope of the possible issue:
+
+- Your name and contact information
+- Description of the vulnerability
+- Attack scenario (if any)
+- Step-by-step instructions to reproduce the issue
+- Any other details
```

Comment:
Should we note here or perhaps elsewhere what to do if this is a generic issue with Substrate/Polkadot-sdk? Or perhaps other dependencies?

Reply:
I added a new line